doc/_static/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
do we really need this empty file?

Reply:
Yes, the folder needs to exist for `sphinx-build` to proceed. But it is temporary, because probably there will be a logo or something that will be added there soon.

Reply:
Hmm, logo. For that we would need someone with design skills.

Reply:
Artists have to survive on something right? :D

But there are things that should be added to at least silent some browsers, like `favicon`, `robots.txt`, etc.

Reply:
I wonder if there is something like ChatGPT which could draw images based on a text description. :-)

Reply:
I mean there are dally and other plenty of "prompt" art. But I don't even trust my artistic eye to identify a child's drawing from a Picasso, so I will deffer on that :D. Still I would rather have something commissioned than research the legalities.